lib/middleman-hashicorp/extension.rb

```diff
@@ -272,6 +272,13 @@ def enterprise_alert(product, variables = {})
       f = File.expand_path("../partials/_enterprise-alert.html.erb", __FILE__)
       render_individual_file(f, variables, { template_body: File.read(f) })
     end
+
+    #
+    # Query the API to get the latest version of a given terraform provider
+    #
+    def latest_provider_version(name)
+      Middleman::HashiCorp::Releases.fetch_latest_version("terraform-provider-#{name}")[:version]
+    end
```

Comment:
To implement the terraform-provider-specific version of the more general latest release fetching method below, we just prefix the name passed in, and return only the version number, making it a nice clean abstraction.